src/components/Rock.js

```diff
@@ -0,0 +1,32 @@
+// import React, { useSelector } from 'react';
```

Comment:
[ OPTIONAL ] Please remove commented code as it can make the whole project unclean.